# update-manager's Finnish translation.
# Copyright (C) 2005-2006 Timo Jyrinki
# This file is distributed under the same license as the update-manager package.
# Timo Jyrinki <timo.jyrinki@iki.fi>, 2005-2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2013-05-22 21:30+0100\n"
"PO-Revision-Date: 2013-05-23 17:33+0000\n"
"Last-Translator: Jiri Grönroos <Unknown>\n"
"Language-Team: Finnish <ubuntu-fi@lists.ubuntu.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2014-04-23 06:51+0000\n"
"X-Generator: Launchpad (build 16985)\n"
"Language: fi\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:211 ../DistUpgrade/distro.py:441
#, python-format
msgid "Server for %s"
msgstr "Palvelin maalle %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:229 ../DistUpgrade/distro.py:235
#: ../DistUpgrade/distro.py:251
msgid "Main server"
msgstr "Pääpalvelin"

#: ../DistUpgrade/distro.py:255
msgid "Custom servers"
msgstr "Määrittele palvelin"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "sources.list-riviä ei voi laskea"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Pakettitiedostoja ei löytynyt. Käytettävä levy ei ehkä ole Ubuntu-levy tai "
"käyttöjärjestelmälaji voi olla väärä."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "CD-levyä ei voitu lisätä"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD-levyä lisättäessä tapahtui virhe, päivitys keskeytyy. Jos käytössä oleva "
"levy on toimiva Ubuntu-CD, tee tästä virheraportti.\n"
"\n"
"Virheilmoitus oli:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:152
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Poista huonossa tilassa oleva paketti"
msgstr[1] "Poista huonossa tilassa olevat paketit"

#: ../DistUpgrade/DistUpgradeCache.py:155
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Paketti ”%s” on ristiriitaisessa tilassa ja se tulee asentaa uudelleen, "
"mutta sen pakettitiedostoa ei löydy. Haluatko poistaa tämän paketin nyt "
"jatkaaksesi?"
msgstr[1] ""
"Paketit ”%s” ovat ristiriitaisessa tilassa ja ne tulee asentaa uudelleen, "
"mutta niiden pakettitiedostoja ei löydy. Haluatko poistaa nämä paketit nyt "
"jatkaaksesi?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:256
msgid "The server may be overloaded"
msgstr "Palvelin saattaa olla ylikuormitettu"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid "Broken packages"
msgstr "Rikkinäisiä paketteja"

#: ../DistUpgrade/DistUpgradeCache.py:370
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Järjestelmä sisältää rikkinäisiä paketteja, joita ei voitu korjata tällä "
"ohjelmalla. Korjaa ne käyttämällä synapticia tai apt-get -komentoa ennen "
"jatkamista."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Päivitystä määriteltäessä tapahtui ratkaisematon ongelma:\n"
"%s\n"
"\n"
" Tämä voi johtua:\n"
" * Päivittämisestä Ubuntun julkaisemattomaan kehitysversioon\n"
" * Ubuntun tämänhetkisen kehitysversion käyttämisestä\n"
" * Ubuntuun kuulumattomien, epävirallisten ohjelmapakettien käyttämisestä\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:648
msgid "This is most likely a transient problem, please try again later."
msgstr "Tämä on luultavasti hetkellinen ongelma. Yritä myöhemmin uudelleen."

#: ../DistUpgrade/DistUpgradeCache.py:651
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Jos mikään näistä ei päde, raportoi aiheesta virheraportti kirjoittamalla "
"päätteeseen seuraava komento:\"ubuntu-bug ubuntu-release-upgrader-core\"."

#: ../DistUpgrade/DistUpgradeCache.py:656
msgid "Could not calculate the upgrade"
msgstr "Tarvittavia päivitykseen liittyviä tarkistuksia ei voitu tehdä"

#: ../DistUpgrade/DistUpgradeCache.py:707
msgid "Error authenticating some packages"
msgstr "Joitain paketteja todennettaessa tapahtui virhe"

#: ../DistUpgrade/DistUpgradeCache.py:708
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Joitain paketteja ei voitu todentaa. Tämä voi olla ohimenevä verkko-ongelma. "
"Voit yrittää myöhemmin uudelleen. Alla on luettelo todentamattomista "
"paketeista."

#: ../DistUpgrade/DistUpgradeCache.py:728
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Paketti \"%s\" on merkitty poistettavaksi, mutta se on poistojen "
"estolistalla."

#: ../DistUpgrade/DistUpgradeCache.py:732
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Pakollinen paketti \"%s\" on merkitty poistettavaksi."

#: ../DistUpgrade/DistUpgradeCache.py:741
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Paketista yritetään asentaa mustalla listalla olevaa versiota \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:859
#, python-format
msgid "Can't install '%s'"
msgstr "Ei voitu asentaa pakettia \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:860
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Vaaditun paketin asennus ei ollut mahdollista. Ilmoita tästä viasta "
"kirjoittamalla päätteeseen \"ubuntu-bug ubuntu-release-upgrader-core\"."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:871
msgid "Can't guess meta-package"
msgstr "Metapakettia ei voitu arvata"

#: ../DistUpgrade/DistUpgradeCache.py:872
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Järjestelmääsi ei ole asennettu ubuntu-desktop-, kubuntu-desktop-, xubuntu-"
"desktop- tai edubuntu-desktop-pakettia, joten käytössä olevaa Ubuntun "
"versiota ei voitu tunnistaa.\n"
"Asenna jokin luetelluista paketeista synapticilla tai apt-get-ohjelmalla "
"ennen jatkamista."

#: ../DistUpgrade/DistUpgradeController.py:126
msgid "Reading cache"
msgstr "Luetaan välimuistia"

#: ../DistUpgrade/DistUpgradeController.py:230
msgid "Unable to get exclusive lock"
msgstr "Ei saatu haluttua lukitusta"

#: ../DistUpgrade/DistUpgradeController.py:231
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Tämä tarkoittaa yleensä, että toinen pakettienhallintaohjelma (kuten apt-get "
"tai aptitude) on jo käynnissä. Sulje tämä toinen ohjelma ensin."

#: ../DistUpgrade/DistUpgradeController.py:289
msgid "Upgrading over remote connection not supported"
msgstr "Päivittäminen etäyhteyden yli ei ole tuettu"

#: ../DistUpgrade/DistUpgradeController.py:290
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Versiopäivitys ssh-yhteyden välityksellä ei ole tuettu. Jos haluat tehdä "
"versiopäivityksen ssh-yhteyden välityksellä, kirjoita komentoriville 'do-"
"release-upgrade'.\n"
"\n"
"Päivitys sulkeutuu nyt. Jos mahdollista, suorita päivitys käyttämättä ssh-"
"yhteyttä."

#: ../DistUpgrade/DistUpgradeController.py:304
msgid "Continue running under SSH?"
msgstr "Jatka käyttäen SSH:ta?"

#: ../DistUpgrade/DistUpgradeController.py:305
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Nykyinen istunto on muodostettu ssh-protokollan välityksellä. Päivitys ssh:n "
"välityksellä ei ole suositeltavaa, sillä mahdollisessa virhetilanteessa "
"palautuminen saattaa olla vaikeaa.\n"
"\n"
"Jos jatkat, ylimääräinen ssh-palvelu avataan porttiin '%s'.\n"
"Haluatko jatkaa?"

#: ../DistUpgrade/DistUpgradeController.py:319
msgid "Starting additional sshd"
msgstr "Käynnistän ylimääräisen SSH-palvelimen"

#: ../DistUpgrade/DistUpgradeController.py:320
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Jotta toipuminen ongelmatilanteesta olisi helpompaa,  käynnistetään porttiin "
"\"%s\" ylimääräinen SSH-palvelin. Jos jokin menee pieleen nykyisen SSH-"
"istunnon kanssa, voit silti yhdistää tähän uuteen.\n"

#: ../DistUpgrade/DistUpgradeController.py:328
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Jos käytössäsi on palomuuri, tämän portin avaaminen väliaikaisesti saattaa "
"olla tarpeen. Koska portin avaus saattaa aiheuttaa tietoturvauhan, porttia "
"ei avata automaattisesti. Voit avata portin seuraavasti:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:400
#: ../DistUpgrade/DistUpgradeController.py:445
msgid "Can not upgrade"
msgstr "Päivitys ei onnistu"

#: ../DistUpgrade/DistUpgradeController.py:401
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Tämä työkalu ei tue päivitystä \"%s\" -> \"%s\"."

#: ../DistUpgrade/DistUpgradeController.py:410
msgid "Sandbox setup failed"
msgstr "Hiekkalaatikon asettaminen epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "It was not possible to create the sandbox environment."
msgstr "Hiekkalaatikkoympäristöä ei ollut mahdollista luoda."

#: ../DistUpgrade/DistUpgradeController.py:417
msgid "Sandbox mode"
msgstr "Hiekkalaatikko-tila"

#: ../DistUpgrade/DistUpgradeController.py:418
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Päivitys ajetaan rajoitetussa hiekkalaatikko- eli testiympäristössä. "
"Muutokset tallennetaan hakemistoon '%s' ja ne katoavat seuraavan "
"uudelleenkäynnistyksen yhteydessä.\n"
"\n"
"*Mitään* muuotoksia ei tallenneta järjestelmähakemistoon tästä hetkestä "
"lähtien seuraavaan uudelleenkäynnistykseen"

#: ../DistUpgrade/DistUpgradeController.py:446
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python-asennuksesi on viallinen. Korjaa symbolinen linkki "
"\"/usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:472
msgid "Package 'debsig-verify' is installed"
msgstr "Paketti \"debsig-verify\" on asennettu"

#: ../DistUpgrade/DistUpgradeController.py:473
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Päivitys ei voi jatkua kyseisen paketin ollessa asennettu.\n"
"Poista paketti Synaptic-ohjelmalla tai komennolla \"apt-get remove debsig-"
"verify\" ja käynnistä päivitys uudelleen."

#: ../DistUpgrade/DistUpgradeController.py:485
#, python-format
msgid "Can not write to '%s'"
msgstr "Ei voida kirjoittaa kohteeseen \"%s\""

#: ../DistUpgrade/DistUpgradeController.py:486
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Hakemistoon '%s' ei voida tehdä muutoksia. Päivitystä ei voida jatkaa.\n"
"Varmista, että kansiolla on kirjoitusoikeudet."

#: ../DistUpgrade/DistUpgradeController.py:497
msgid "Include latest updates from the Internet?"
msgstr "Sisällytä uusimmat päivitykset Internetistä?"

#: ../DistUpgrade/DistUpgradeController.py:498
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Päivitysjärjestelmä voi ladata Internetistä uusimmat päivitykset "
"automaattisesti ja asentaa ne päivityksen aikana. Jos sinulla on "
"verkkoyhteys käytettävissä, tämä on hyvin suositeltavaa.\n"
"\n"
"Päivitys kestää tällöin pidempään, mutta valmistuttuaan järjestelmä on "
"täysin ajan tasalla. Jos et tee tätä nyt, uusimmat päivitykset tulee joka "
"tapauksessa ladata pian päivityksen jälkeen.\n"
"Jos vastaat \"ei\" tähän, verkkoa ei käytetä lainkaan."

#: ../DistUpgrade/DistUpgradeController.py:729
#, python-format
msgid "disabled on upgrade to %s"
msgstr "otettu pois käytöstä päivitettäessä versioon %s"

#: ../DistUpgrade/DistUpgradeController.py:762
msgid "No valid mirror found"
msgstr "Sopivaa peilipalvelinta ei löytynyt"

#: ../DistUpgrade/DistUpgradeController.py:753
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Ohjelmalähteen tietoja luettaessa ei löytynyt merkintää päivityksestä. Tämä "
"voi tapahtua jos käytössäsi on yksityinen ohjelmalähde tai ohjelmalähteen "
"tiedot eivät ole ajan tasalla.\n"
"\n"
"Korvataanko \"sources.list\"-tiedosto joka tapauksessa? Jos valitset "
"\"Kyllä\", jokainen \"%s\" muutetaan muotoon \"%s\"."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:783
msgid "Generate default sources?"
msgstr "Lisätäänkö oletusohjelmalähteet?"

#: ../DistUpgrade/DistUpgradeController.py:784
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"”sources.list”-tiedostosta ei löytynyt kelvollista riviä lähteelle ”%s”.\n"
"\n"
"Tulisiko lähteen ”%s” oletusrivit lisätä? Jos valitse ”Ei”, päivitys "
"keskeytyy."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Repository information invalid"
msgstr "Virhe ohjelmalähdetiedoissa"

#: ../DistUpgrade/DistUpgradeController.py:820
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Ohjelmistolähdetietojen päivitys tuotti virheellisen tiedoston, joten "
"vianilmoitusprosessi käynnistetään."

#: ../DistUpgrade/DistUpgradeController.py:827
msgid "Third party sources disabled"
msgstr "Kolmannen osapuolen ohjelmalähteet poissa käytöstä"

#: ../DistUpgrade/DistUpgradeController.py:828
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Jotkin kolmannen osapuolen lähteet sources.list-tiedostossa ovat nyt poissa "
"käytöstä. Voit ottaa ne uudelleen käyttöön päivityksen jälkeen "
"\"Ohjelmalähteet\"-työkalulla tai pakettienhallintaohjelmalla."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paketti ristiriitaisessa tilassa"
msgstr[1] "Paketteja ristiriitaisessa tilassa"

#: ../DistUpgrade/DistUpgradeController.py:873
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Paketti ”%s” on ristiriitaisessa tilassa ja se tulee asentaa uudelleen, "
"mutta sen pakettitiedostoa ei löydy. Asenna paketti uudelleen käsin tai "
"poista se järjestelmästä."
msgstr[1] ""
"Paketit ”%s” ovat ristiriitaisessa tilassa ja ne tulee asentaa uudelleen, "
"mutta niiden pakettitiedostoja ei löydy. Asenna paketit uudelleen käsin tai "
"poista ne järjestelmästä."

#: ../DistUpgrade/DistUpgradeController.py:921
msgid "Error during update"
msgstr "Virhe päivitettäessä"

#: ../DistUpgrade/DistUpgradeController.py:922
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Päivitettäessä tapahtui virhe. Tämä on yleensä jonkinlainen verkko-ongelma. "
"Tarkista verkkoyhteytesi toiminta ja yritä uudelleen."

#: ../DistUpgrade/DistUpgradeController.py:931
msgid "Not enough free disk space"
msgstr "Levytilaa ei ole riittävästi"

#: ../DistUpgrade/DistUpgradeController.py:932
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Päivitys on keskeytynyt. Päivitys vaatii yhteensä %s vapaata tilaa levyllä "
"”%s”. Vapauta siis vähintään %s lisää tilaa levyllä ”%s”. Tyhjennä roskakori "
"sekä poista aiempien asennusten väliaikaiset tiedostot komennolla ”sudo apt-"
"get clean”."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:961
#: ../DistUpgrade/DistUpgradeController.py:1743
msgid "Calculating the changes"
msgstr "Lasketaan muutoksia"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:993
msgid "Do you want to start the upgrade?"
msgstr "Haluatko aloittaa päivityksen?"

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid "Upgrade canceled"
msgstr "Päivitys peruttiin"

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Päivitys perutaan ja järjestelmän alkuperäinen tila palautetaan. Voit jatkaa "
"päivitystä halutessasi myöhemmin."

#: ../DistUpgrade/DistUpgradeController.py:1073
#: ../DistUpgrade/DistUpgradeController.py:1197
msgid "Could not download the upgrades"
msgstr "Päivityksiä ei voitu noutaa"

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Päivitys on keskeytynyt. Tarkista Internet-yhteytesi tai asennusmedia ja "
"yritä uudelleen. Kaikki ladatut tiedostot on toistaiseksi pidetty."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1148
#: ../DistUpgrade/DistUpgradeController.py:1185
#: ../DistUpgrade/DistUpgradeController.py:1290
msgid "Error during commit"
msgstr "Virhe suoritettaessa"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1150
#: ../DistUpgrade/DistUpgradeController.py:1187
#: ../DistUpgrade/DistUpgradeController.py:1329
msgid "Restoring original system state"
msgstr "Palautetaan alkuperäistä järjestelmän tilaa"

#: ../DistUpgrade/DistUpgradeController.py:1151
#: ../DistUpgrade/DistUpgradeController.py:1166
#: ../DistUpgrade/DistUpgradeController.py:1188
msgid "Could not install the upgrades"
msgstr "Päivityksiä ei voitu asentaa"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1156
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Päivitys on keskeytynyt. Järjestelmä saattaa olla epävakaassa tilassa. "
"Palautuskomento suoritetaan nyt (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1161
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Ilmoita tästä viasta menemällä selaimella osoitteeseen "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug ja "
"liittämällä polussa /var/log/dist-upgrade/ olevat tiedostot "
"vikailmoitukseen.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1198
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Päivitys on keskeytynyt. Tarkista Internet-yhteytesi tai asennusmediasi ja "
"yritä uudelleen. "

#: ../DistUpgrade/DistUpgradeController.py:1278
msgid "Remove obsolete packages?"
msgstr "Poistetaanko vanhentuneet paketit?"

#: ../DistUpgrade/DistUpgradeController.py:1279
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Säilytä"

#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "_Remove"
msgstr "_Poista"

#: ../DistUpgrade/DistUpgradeController.py:1291
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Siistimisvaiheessa ilmeni ongelma. Lisätietoja alla olevassa viestissä. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1367
msgid "Required depends is not installed"
msgstr "Vaadittuja riippuvuuksia ei ole asennettu"

#: ../DistUpgrade/DistUpgradeController.py:1368
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Vaadittu riippuvuus \"%s\" ei ole asennettu. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1701
msgid "Checking package manager"
msgstr "Tarkistetaan pakettienhallintaa"

#: ../DistUpgrade/DistUpgradeController.py:1641
msgid "Preparing the upgrade failed"
msgstr "Päivityksen valmistelu epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:1642
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Järjestelmän valmistelu versiopäivitystä varten epäonnistui, joten "
"vianilmoitusprosessi käynnistetään."

#: ../DistUpgrade/DistUpgradeController.py:1656
msgid "Getting upgrade prerequisites failed"
msgstr "Päivityksen esivaatimusten nouto epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:1657
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Päivityksen esivaatimuksia ei saatu, ja päivitys lopetetaan. Järjestelmä "
"palautetaan alkuperäiseen tilaansa.\n"
"\n"
"Virheraportointityökalu käynnistetään."

#: ../DistUpgrade/DistUpgradeController.py:1685
msgid "Updating repository information"
msgstr "Päivitetään ohjelmalähdetietoja"

#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Failed to add the cdrom"
msgstr "CD-levyn lisääminen ei onnistunut"

#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Sorry, adding the cdrom was not successful."
msgstr "CD-levyn lisääminen ei onnistunut"

#: ../DistUpgrade/DistUpgradeController.py:1724
msgid "Invalid package information"
msgstr "Pakettitiedot viallisia"

#: ../DistUpgrade/DistUpgradeController.py:1725
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1749
#: ../DistUpgrade/DistUpgradeController.py:1805
msgid "Fetching"
msgstr "Noudetaan"

#: ../DistUpgrade/DistUpgradeController.py:1755
#: ../DistUpgrade/DistUpgradeController.py:1809
msgid "Upgrading"
msgstr "Päivitetään"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1760
#: ../DistUpgrade/DistUpgradeController.py:1811
#: ../DistUpgrade/DistUpgradeController.py:1818
#: ../DistUpgrade/DistUpgradeController.py:1829
msgid "Upgrade complete"
msgstr "Päivitys on valmis"

#: ../DistUpgrade/DistUpgradeController.py:1761
#: ../DistUpgrade/DistUpgradeController.py:1812
#: ../DistUpgrade/DistUpgradeController.py:1819
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Päivitys valmistui, mutta päivityksen yhteydessä tapahtui virheitä."

#: ../DistUpgrade/DistUpgradeController.py:1768
msgid "Searching for obsolete software"
msgstr "Etsitään vanhentuneita ohjelmistoja"

#: ../DistUpgrade/DistUpgradeController.py:1781
msgid "System upgrade is complete."
msgstr "Järjestelmän päivitys on valmis."

#: ../DistUpgrade/DistUpgradeController.py:1830
msgid "The partial upgrade was completed."
msgstr "Osittainen päivitys valmistui."

#: ../DistUpgrade/DistUpgradeFetcher.py:123
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:119
msgid "Could not find the release notes"
msgstr "Julkaisutietoja ei löytynyt"

#: ../DistUpgrade/DistUpgradeFetcher.py:124
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:120
msgid "The server may be overloaded. "
msgstr "Palvelin voi olla ylikuormitettu. "

#: ../DistUpgrade/DistUpgradeFetcher.py:136
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:124
msgid "Could not download the release notes"
msgstr "Julkaisutietoja ei voitu noutaa"

#: ../DistUpgrade/DistUpgradeFetcher.py:137
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:125
msgid "Please check your internet connection."
msgstr "Tarkista Internet-yhteytesi toimivuus."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "todentaa '%(file)s' vastaan '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#, python-format
msgid "extracting '%s'"
msgstr "puretaan \"%s\""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:157
#: ../DistUpgrade/DistUpgradeFetcherCore.py:158
msgid "Could not run the upgrade tool"
msgstr "Päivitystyökalua ei voitu suorittaa"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:159
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Tämä johtuu todennäköisesti päivitysohjelmistossa olevasta ongelmasta. Tee "
"vikailmoitus komennolla \"ubuntu-bug ubuntu-release-upgrader-core\"."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:236
msgid "Upgrade tool signature"
msgstr "Päivitystyökalun allekirjoitus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Upgrade tool"
msgstr "Päivitystyökalu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to fetch"
msgstr "Nouto epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Päivityksen noutaminen epäonnistui. Tämä voi johtua verkko-ongelmasta. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:282
msgid "Authentication failed"
msgstr "Todennus epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Päivityksen todennus epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:288
msgid "Failed to extract"
msgstr "Purku epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:289
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Päivityksen purkaminen epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:294
msgid "Verification failed"
msgstr "Varmennus epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:295
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Päivityksen tarkistaminen epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:309
#: ../DistUpgrade/DistUpgradeFetcherCore.py:315
msgid "Can not run the upgrade"
msgstr "Päivitystä ei voi suorittaa"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:310
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Tämä johtuu yleensä siitä, että /tmp on liitetty noexec-valintaa käyttäen. "
"Liitä /tmp uudelleen ilman noexec-valintaa ja suorita päivitys uudelleen."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:316
#, python-format
msgid "The error message is '%s'."
msgstr "Virheviesti oli \"%s\"."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:70
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:101
msgid "Upgrade"
msgstr "Päivitä"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:105
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Julkaisutiedot"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:147
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:160
msgid "Downloading additional package files..."
msgstr "Noudetaan tarvittavia pakettitiedostoja..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:162
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Tiedosto %s/%s nopeudella %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:166
#, python-format
msgid "File %s of %s"
msgstr "Tiedosto %s/%s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:173
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:115
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Aseta \"%s\" asemaan \"%s\""

#. change = QMessageBox.question(None, _("Media Change"), msg,
#. QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:177
#: ../DistUpgrade/DistUpgradeViewKDE.py:197
msgid "Media Change"
msgstr "Median vaihto"

#: ../DistUpgrade/DistUpgradeQuirks.py:208
msgid "evms in use"
msgstr "evms on käytössä"

#: ../DistUpgrade/DistUpgradeQuirks.py:209
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"/proc/mounts -tiedoston mukaan järjestelmässäsi on käytössä evms-"
"taltionhallinta. evms-ohjelmistoa ei enää tueta, joten ole hyvä ja poista se "
"käytöstä ja suorita päivitys uudelleen."

#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Your graphics hardware may not be fully supported in Ubuntu 13.04."
msgstr ""
"Näytönohjaimesi ei välttämättä ole täysin tuettu Ubuntun 13.04-versiossa."

#: ../DistUpgrade/DistUpgradeQuirks.py:248
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""
"Unity-työpöytäympäristö ei ole täysin tuettu näytönohjaimellasi. Saatat "
"huomata päivityksen jälkeen järjestelmän olevan erittäin hidas. "
"Suosittelemme pitäytymään pitkäaikaisesti tuetussa LTS-versiossa "
"toistaiseksi. Lue lisätietoja osoitteessa "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D - Haluatko "
"kaikesta huolimatta jatkaa päivitystä?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Näytönohjaimesi ei välttämättä ole täysin tuettu Ubuntu 12.04 LTS -versiossa."

#: ../DistUpgrade/DistUpgradeQuirks.py:274
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Intel-näytönohjaimesi tuki on rajoittunut Ubuntu 12.04 LTS -versiossa, joten "
"saatat havaita ongelmia päivityksen jälkeen. Lisätietoja englanniksi: "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx - Haluatko jatkaa "
"päivitystä?"

#: ../DistUpgrade/DistUpgradeQuirks.py:294
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Päivitys saattaa poistaa käytöstä visuaalisia tehosteita ja huonontaa "
"joidenkin pelien ja graafisesti raskaiden ohjelmien suorituskykyä."

#: ../DistUpgrade/DistUpgradeQuirks.py:564
#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tämä tietokone käyttää tällä hetkellä NVIDIAn ”nvidia”-näyttöajuria. "
"Näytönohjaimesi kanssa toimivaa versiota tästä ajurista ei ole saatavilla "
"Ubuntu 10.04 LTS:lle.\n"
"\n"
"Haluatko jatkaa?"

#: ../DistUpgrade/DistUpgradeQuirks.py:298
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tämä tietokone käyttää tällä hetkellä AMD:n ”fglrx”-näyttöajuria. "
"Näytönohjaimesi kanssa toimivaa versiota tästä ajurista ei ole saatavilla "
"Ubuntu 10.04 LTS:lle.\n"
"\n"
"Haluatko jatkaa?"

#: ../DistUpgrade/DistUpgradeQuirks.py:333
msgid "No i686 CPU"
msgstr "Ei i686-suoritin"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Järjestelmäsi käyttää i586-suoritinta tai muuta suoritinta, jossa ei ole "
"cmov-laajennusta. Kaikki paketit on käännetty asetuksin, jotka vaativat "
"vähintään i686-suorittimen. Nykyistä laitteistoasi ei siis ole mahdollista "
"päivittää uudempaan Ubuntu-versioon."

#: ../DistUpgrade/DistUpgradeQuirks.py:370
msgid "No ARMv6 CPU"
msgstr "Ei ARMv6-suoritinta"

#: ../DistUpgrade/DistUpgradeQuirks.py:371
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Järjestelmässäsi on ARM-suoritin jonka arkkitehtuuri on vanhempi kuin ARMv6. "
"Kaikki Karmicin ohjelmapaketit on optimoitu siten, että ne vaativat "
"vähintään ARMv6-arkkitehtuuria tukevan suorittimen. Tästä syystä "
"järjestelmäsi päivittäminen uuteen Ubuntun julkaisuun ei ole mahdollista."

#: ../DistUpgrade/DistUpgradeQuirks.py:391
msgid "No init available"
msgstr "Init-palvelua ei löydy"

#: ../DistUpgrade/DistUpgradeQuirks.py:392
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Järjestelmäsi vaikuttaisi toimivan virtualisoidussa ympäristössä, jossa ei "
"ole init-taustaohjelmaa. Tällaisia ovat esimerkiksi Linux-VServer. Ubuntu "
"10.04 LTS ei toimi tällaisessa ympäristössä, vaan vaatii ensin "
"virtuaalikoneen asetusten päivitystä.\n"
"\n"
"Haluatko varmasti jatkaa?"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Sandbox upgrade using aufs"
msgstr "Hiekkalaatikkopäivitys aufs:ää käyttäen"

#: ../DistUpgrade/DistUpgradeMain.py:66
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Käytä annettua polkua päivitettyjen pakettien etsimiseksi CD-levyltä."

#: ../DistUpgrade/DistUpgradeMain.py:72
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Valitse käyttöliittymä. Tällä hetkellä valittavissa ovat: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "*DEPRECATED* this option will be ignored"
msgstr "*VANHENNETTU* tämä valitsin ohitetaan"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Suorita vain osittainen päivitys (ei sources.list-uudelleenkirjoitusta)"

#: ../DistUpgrade/DistUpgradeMain.py:81
msgid "Disable GNU screen support"
msgstr "Poista käytöstä GNU screen -tuki"

#: ../DistUpgrade/DistUpgradeMain.py:83
msgid "Set datadir"
msgstr "Aseta datahakemisto"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:136
#: ../DistUpgrade/DistUpgradeViewKDE.py:210
msgid "Fetching is complete"
msgstr "Nouto on valmis"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk3.py:151
#: ../DistUpgrade/DistUpgradeViewKDE.py:226
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Noudetaan tiedostoa %li/%li nopeudella %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:151
#: ../DistUpgrade/DistUpgradeViewGtk.py:298
#: ../DistUpgrade/DistUpgradeViewGtk3.py:153
#: ../DistUpgrade/DistUpgradeViewGtk3.py:310
#: ../DistUpgrade/DistUpgradeViewKDE.py:227
#: ../DistUpgrade/DistUpgradeViewKDE.py:375
#, python-format
msgid "About %s remaining"
msgstr "Noin %s jäljellä"

#: ../DistUpgrade/DistUpgradeViewGtk.py:154
#: ../DistUpgrade/DistUpgradeViewGtk3.py:156
#: ../DistUpgrade/DistUpgradeViewKDE.py:229
#, python-format
msgid "Fetching file %li of %li"
msgstr "Noudetaan tiedostoa %li/%li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:185
#: ../DistUpgrade/DistUpgradeViewGtk3.py:187
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "Applying changes"
msgstr "Muutoksia toteutetaan"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:210
#: ../DistUpgrade/DistUpgradeViewGtk3.py:213
#: ../DistUpgrade/DistUpgradeViewKDE.py:279
msgid "dependency problems - leaving unconfigured"
msgstr "riippuvuusongelmia - jätetään asetukset säätämättä"

#: ../DistUpgrade/DistUpgradeViewGtk.py:215
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:281
#, python-format
msgid "Could not install '%s'"
msgstr "Pakettia \"%s\" ei voitu asentaa"

#: ../DistUpgrade/DistUpgradeViewGtk.py:216
#: ../DistUpgrade/DistUpgradeViewGtk3.py:219
#: ../DistUpgrade/DistUpgradeViewKDE.py:282
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Päivitys jatkuu mutta paketti %s ei välttämättä toimi päivityksen jälkeen. "
"Harkitse vikailmoituksen tekemistä virheestä."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:233
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Korvataanko alkuperäisestä muutettu asetustiedosto\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:234
#: ../DistUpgrade/DistUpgradeViewGtk3.py:237
#: ../DistUpgrade/DistUpgradeViewKDE.py:304
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Tähän asetustiedostoon tehdyt muutokset menetetään, jos se korvataan "
"uudemmalla versiolla."

#: ../DistUpgrade/DistUpgradeViewGtk.py:253
#: ../DistUpgrade/DistUpgradeViewGtk3.py:257
#: ../DistUpgrade/DistUpgradeViewKDE.py:327
msgid "The 'diff' command was not found"
msgstr "Komentoa 'diff' ei löytynyt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:466
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
#: ../DistUpgrade/DistUpgradeViewText.py:100
msgid "A fatal error occurred"
msgstr "Tapahtui vakava virhe"

#: ../DistUpgrade/DistUpgradeViewGtk.py:467
#: ../DistUpgrade/DistUpgradeViewGtk3.py:479
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ilmoita tästä virheraportilla (jos et ole niin jo tehnyt) ja sisällytä "
"siihen tiedostot /var/log/dist-upgrade/main.log ja /var/log/dist-"
"upgrade/apt.log. Päivitys on keskeytynyt.\n"
"Alkuperäinen sources.list tallennettiin nimellä "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:484
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid "Ctrl-c pressed"
msgstr "Ctrl-c painettu"

#: ../DistUpgrade/DistUpgradeViewGtk.py:485
#: ../DistUpgrade/DistUpgradeViewGtk3.py:497
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Tämä keskeyttää toiminnon ja saattaa jättää järjestelmän rikkinäiseen "
"tilaan. Haluatko varmasti tehdä sen?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:633
#: ../DistUpgrade/DistUpgradeViewGtk3.py:630
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Jottei tietoja häviäisi, sulje kaikki avoinna olevat ohjelmat ja asiakirjat."

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Canonical ei enää tue seuraavia (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Varhenna (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "Remove (%s)"
msgstr "Poista (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "No longer needed (%s)"
msgstr "Ei enää tarpeellinen (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:651
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Install (%s)"
msgstr "Asenna (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:652
#: ../DistUpgrade/DistUpgradeViewGtk3.py:649
#, python-format
msgid "Upgrade (%s)"
msgstr "Päivitä (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:339
msgid "Show Difference >>>"
msgstr "Näytä erot >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:342
msgid "<<< Hide Difference"
msgstr "<<< Piilota erot"

#: ../DistUpgrade/DistUpgradeViewKDE.py:558
msgid "Error"
msgstr "Virhe"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
msgid "&Cancel"
msgstr "&Peru"

#: ../DistUpgrade/DistUpgradeViewKDE.py:817
msgid "&Close"
msgstr "&Sulje"

#: ../DistUpgrade/DistUpgradeViewKDE.py:622
msgid "Show Terminal >>>"
msgstr "Näytä pääte >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:625
msgid "<<< Hide Terminal"
msgstr "<<< Piilota pääte"

#: ../DistUpgrade/DistUpgradeViewKDE.py:705
msgid "Information"
msgstr "Tiedot"

#: ../DistUpgrade/DistUpgradeViewKDE.py:755
#: ../DistUpgrade/DistUpgradeViewKDE.py:800
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Yksityiskohdat"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#, python-format
msgid "No longer supported %s"
msgstr "Ei enää tuettu %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Remove %s"
msgstr "Poista %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#: ../DistUpgrade/DistUpgradeViewText.py:194
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Poistetaan (oli automaattisesti asennettu) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:787
#, python-format
msgid "Install %s"
msgstr "Asenna %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:789
#, python-format
msgid "Upgrade %s"
msgstr "Päivitä %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
#: ../DistUpgrade/DistUpgradeViewText.py:242
msgid "Restart required"
msgstr "Uudelleenkäynnistys vaaditaan"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Käynnistä järjestelmä uudelleen päivityksen "
"viimeistelemiseksi</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:816
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Uudelleenkäynnistä nyt"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Peru käynnissä oleva päivitys?</big></b>\n"
"\n"
"Järjestelmä voi olla käyttökelvottomassa tilassa, jos perut päivityksen. "
"Päivityksen jatkaminen on erittäin suositeltavaa."

#: ../DistUpgrade/DistUpgradeViewKDE.py:838
msgid "Cancel Upgrade?"
msgstr "Peru päivitys?"

#: ../DistUpgrade/DistUpgradeView.py:73
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li päivä"
msgstr[1] "%li päivää"

#: ../DistUpgrade/DistUpgradeView.py:75
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li tunti"
msgstr[1] "%li tuntia"

#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuutti"
msgstr[1] "%li minuuttia"

#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekunti"
msgstr[1] "%li sekuntia"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:94
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:112
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:163
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Nouto kestää noin %s 1Mbit-DSL-yhteydellä ja noin %s 56kbit-modeemilla."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:167
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Nouto kestää yhteydelläsi noin %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:271
#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Valmistellaan päivitystä"

#: ../DistUpgrade/DistUpgradeView.py:272
msgid "Getting new software channels"
msgstr "Noudetaan uusia ohjelmalähteitä"

#: ../DistUpgrade/DistUpgradeView.py:273
#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Haetaan uusia paketteja"

#: ../DistUpgrade/DistUpgradeView.py:274
#: ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Asennetaan päivityksiä"

#: ../DistUpgrade/DistUpgradeView.py:275
#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Siistitään"

#: ../DistUpgrade/DistUpgradeView.py:360
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"Canonical ei enää tue %(amount)d asennettua pakettia. Voit edelleen "
"vastaanottaa yhteisön tarjoaman tuen."
msgstr[1] ""
"Canonical ei enää tue %(amount)d asennettua pakettia. Voit edelleen "
"vastaanottaa yhteisön tarjoaman tuen."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:369
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d paketti tullaan poistamaan."
msgstr[1] "%d pakettia tullaan poistamaan."

#: ../DistUpgrade/DistUpgradeView.py:374
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d uusi paketti asennetaan."
msgstr[1] "%d uutta pakettia asennetaan."

#: ../DistUpgrade/DistUpgradeView.py:380
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d paketti päivitetään."
msgstr[1] "%d pakettia päivitetään."

#: ../DistUpgrade/DistUpgradeView.py:388
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Noudettavaa yhteensä %s. "

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Versiopäivityksen asennus saattaa kestää useita tunteja. Kun lataus on "
"valmis, prosessia ei ole mahdollista perua."

#: ../DistUpgrade/DistUpgradeView.py:397
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Päivitysten lataaminen ja asentaminen voi kestää useita tunteja. Lataamisen "
"valmistuttua päivitystä ei voi peruuttaa."

#: ../DistUpgrade/DistUpgradeView.py:402
msgid "Removing the packages can take several hours. "
msgstr "Pakettien poisto saattaa kestää useita tunteja. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:407
msgid "The software on this computer is up to date."
msgstr "Tämän tietokoneen ohjelmistot on päivitetty."

#: ../DistUpgrade/DistUpgradeView.py:408
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Päivityksiä ei ole saatavilla järjestelmälle. Päivitys keskeytyy."

#: ../DistUpgrade/DistUpgradeView.py:421
msgid "Reboot required"
msgstr "Uudelleenkäynnistys vaaditaan"

#: ../DistUpgrade/DistUpgradeView.py:422
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Päivitys on valmis ja uudelleenkäynnistys vaaditaan. Haluatko käynnistää "
"tietokoneen uudelleen nyt?"

#: ../DistUpgrade/DistUpgradeViewText.py:101
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ilmoita tästä virheraportilla ja sisällytä siihen tiedostot /var/log/dist-"
"upgrade/main.log ja /var/log/dist-upgrade/apt.log. Päivitys on keskeytynyt.\n"
"Alkuperäinen sources.list tallennettiin nimellä "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:125
msgid "Aborting"
msgstr "Peruutetaan"

#: ../DistUpgrade/DistUpgradeViewText.py:130
msgid "Demoted:\n"
msgstr "Alennettu:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "To continue please press [ENTER]"
msgstr "Jatka painamalla Enter"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
#: ../DistUpgrade/DistUpgradeViewText.py:215
msgid "Continue [yN] "
msgstr "Jatka [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "Details [d]"
msgstr "Yksityiskohdat [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:174
#: ../DistUpgrade/DistUpgradeViewText.py:218
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:177
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:180
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ei enää tuettu: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Remove: %s\n"
msgstr "Poista: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:200
#, python-format
msgid "Install: %s\n"
msgstr "Asenna: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
#, python-format
msgid "Upgrade: %s\n"
msgstr "Päivitä: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:222
msgid "Continue [Yn] "
msgstr "Jatka [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:243
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Uudelleenkäynnistys vaaditaan päivityksen saattamiseksi loppuun.\n"
"Jos valitset \"y\" (\"yes\" eli \"kyllä\") järjestelmä käynnistetään "
"uudelleen."

#: ../DistUpgrade/GtkProgress.py:73
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Noudetaan tiedostoa %(current)li/%(total)li nopeudella %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:79
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Noudetaan tiedostoa %(current)li/%(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Näytä tiedostojen edistyminen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Peru päivitys"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Jatka päivitystä"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Peruuta käynnissä oleva päivitys?</big></b>\n"
"\n"
"Järjestelmä voi olla käyttökelvottomassa tilassa, jos peruutat päivityksen. "
"Päivityksen jatkaminen on erittäin suositeltavaa."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Aloita päivitys"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Korvaa"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Tiedostojen väliset erot"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Tee virheraportti"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Jatka"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Aloitetaanko päivitys?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Käynnistä järjestelmä uudelleen, jotta päivitys valmistuu</big></b>\n"
"\n"
"Tallenna työsi ennen jatkamista."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Jakelupäivitys"

#: ../data/gtkbuilder/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 13.04</big></b>"
msgstr "<b><big>Päivitetään Ubuntu versioon 13.04</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Asetetaan uusia ohjelmalähteitä"

#: ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Käynnistetään tietokone uudelleen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Pääte"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Päivitä"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Uusi Ubuntun versio on saatavilla. Haluatko aloittaa päivityksen?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Älä päivitä"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Kysy myöhemmin uudestaan"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Kyllä, päivitä nyt"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Valitsin päivittämisen uuteen Ubuntun versioon"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Päivitys voidaan suorittaa myös myöhemmin avaamalla Ohjelmistopäivitykset ja "
"napsauttamalla \"Päivitä\"."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Suorita järjestelmäpäivitys"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Ubuntun versiopäivitys vaatii tunnistautumisen."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Suorita osittainen päivitys"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Osittainen päivitys vaatii tunnistautumisen."

#: ../do-partial-upgrade:67 ../do-release-upgrade:69
msgid "Show version and exit"
msgstr "Näytä versio ja poistu"

#: ../do-partial-upgrade:70 ../do-release-upgrade:76
msgid "Directory that contains the data files"
msgstr "Datatiedostoja sisältävä hakemisto"

#: ../do-partial-upgrade:73 ../do-release-upgrade:89
msgid "Run the specified frontend"
msgstr "Aja määritetty edustaohjelma"

#: ../do-partial-upgrade:90
msgid "Running partial upgrade"
msgstr "Suoritetaan osittaista päivitystä"

#: ../do-release-upgrade:31
msgid "Downloading the release upgrade tool"
msgstr "Noudetaan julkaisupäivitystyökalua"

#: ../do-release-upgrade:72 ../check-new-release-gtk:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Tarkista onko päivittäminen uusimpaan kehitysversioon mahdollista"

#: ../do-release-upgrade:79 ../check-new-release-gtk:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Yritä päivittää uusimpaan julkaisuun käyttäen päivitysohjelmaa kohteesta "
"$distro-proposed"

#: ../do-release-upgrade:83
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Suorita erityisessä päivitystilassa.\n"
"Tällä hetkellä tuettuina ovat \"desktop\" työpöytäjärjestelmän ja \"server\" "
"palvelinjärjestelmän päivittämiseen."

#: ../do-release-upgrade:91
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Testaa päivitystä hiekkalaatikossa aufs-peitekerroksella"

#: ../do-release-upgrade:94
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Tarkista vain onko uusi jakelujulkaisu saatavilla, ja kerro tulos "
"poistumiskoodilla"

#: ../do-release-upgrade:108
msgid "Checking for a new Ubuntu release"
msgstr "Etsitään uutta Ubuntu-julkaisua"

#: ../do-release-upgrade:121
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ubuntu-versiosi ei ole enää tuettu."

#: ../do-release-upgrade:122
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Lisätietoja päivityksestä osoitteessa\n"
"%(url)s\n"

#: ../do-release-upgrade:128
msgid "No new release found"
msgstr "Uutta julkaisua ei löytynyt"

#: ../do-release-upgrade:133
msgid "Release upgrade not possible right now"
msgstr "Julkaisupäivitystä ei ole mahdollista suorittaa juuri nyt"

#: ../do-release-upgrade:134
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Julkaisupäivitystä ei voi suorittaa juuri nyt, koeta myöhemmin uudelleen. "
"Palvelimen vastaus: '%s'"

#: ../do-release-upgrade:140
#, c-format
msgid "New release '%s' available."
msgstr "Uusi julkaisu ”%s” saatavilla."

#: ../do-release-upgrade:141
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Aja \"do-release-upgrade\" päivittääksesi siihen."

#: ../check-new-release-gtk:113
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Päivitys versioon Ubuntu %(version)s on saatavilla"

#: ../check-new-release-gtk:144
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Voit valita päivityksen Ubuntun versioon %s"

#: ../check-new-release-gtk:186
msgid "Add debug output"
msgstr "Lisää virheenjäljitystiedot"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Järjestelmäpäivityksen suorittaminen vaatii tunnistautumisen"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Osittaisen päivityksen suorittaminen vaatii tunnistautumisen"
